android/AndroidManifest.xml

```diff
@@ -66,12 +63,14 @@
       android:label="@string/app_name"
       android:localeConfig="@xml/locales_config"
       android:theme="@style/MwmTheme"
+      android:resizeableActivity="true"
       android:supportsRtl="true"
       android:networkSecurityConfig="@xml/network_security_config"
       tools:targetApi="t">
 
     <activity
       android:name="app.organicmaps.SplashActivity"
+      android:configChanges="orientation|screenSize|smallestScreenSize|density|screenLayout|uiMode|keyboard|keyboardHidden|navigation"
```

Comment:
Isn't it a dangerous change? All these events may not be handled in the code. Why is it needed for the SplashActivity?

Reply:
No this is not dangerous. It just tells the system to not restart the activity, but to receive a call to the onConfigurationChanged(Configuration) method along with the new configuration. It saves us from writing a bunch of code to let the processes live when the orientation expectedly changes.

Reply:
Why is it needed in the first place in the SplashActivity? This activity's role is to properly pass data to other activities and to finish the startup ASAP.

Reply:
The official documentation suggests adding this to the main activity of the application. I assumed that the SplashActivity is the correct one for OM, but I can correct this if I am wrong here.

We are just trying to guarantee that unexpected configuration changes don't crash the app.

Reply:
1. Why does the official documentation suggest adding it? What exactly will work/won't work with/without this change?
2. Is this line change really required in our case?
3. Did you test it on real devices?